docs/_navbar.md

```diff
@@ -1,6 +1,6 @@
 - Translations
   - [:us: en-us](/)
-  - [:fr: fr-fr](/fr/)
+  - [:fr: fr](/fr/)
```

Comment:
```suggestion
  - [:fr: fr-fr](/fr/)
```

Can we leave it for now to be consistent and we can adjust this in a separate PR?